### PR TITLE
Added 6sense v1.0.0 trajectories handling.

### PR DESCRIPTION
**Pull request by 4rzael**
## Also:

* Generalized the way maestro handle measurement storing (now using json)

* Added raw measurement API

* Updated dependencies: pheromon-codecs v0.0.4

* Added (and fixed) some tests

* Cleaned the code (0 eslint errors/warning)

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -262,27 +264,28 @@ var toExport = {
...
         },
+
         getPlaceMeasurements: function(ids, types, start, end){
             return databaseP.then(function(db){
 
                 // if no dates provided, assume we want all
-                var start = start ? start : new Date("1900-10-15T11:23:19.766Z");
-                var end = end ? end : new Date("2200-10-15T11:23:19.766Z");
+                var _start = start || new Date('1900-10-15T11:23:19.766Z');
+                var _end = end || new Date('2200-10-15T11:23:19.766Z');
```
Comment (vallettea) [inline database/index.js]: tricky error I did corrected here by @4rzael : when you write var a, a is undefined and thus the ? test alwas falls back on the second choice

File: database/models/measurements.js [hunk 2]
```diff
@@ -10,8 +10,12 @@ module.exports = {
     create: function (data) {
         return databaseP.then(function (db) {
 
+        var cloneData = JSON.parse(JSON.stringify(data)); // Copy the object...
```
Comment (vallettea) [inline database/models/measurements.js]: @4rzael used this to copy an object, which works in fact. 
@DavidBruant is this ok for you ?

**Comment by vallettea** [comment]
:clap:

File: database/models/measurements.js (hunk 2 above)
Comment (DavidBruant) [inline database/models/measurements.js]: Yep sounds good. Not to be done too often. More and more we should consider using immutable-js for more cases.

Comment (iOiurson) [inline database/models/measurements.js]: If i remember correctly, Object.assign() was used instead.

Comment (DavidBruant) [inline database/models/measurements.js]: Oh right! There is another way to clone an object doing;
```js
var clone = Object.assign({}, obj);
```
This is probably much more efficient than doing it via JSON.

**Pull request merged**